src/Pages/CoinsList.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import millify from 'millify';
+import { useSelector } from 'react-redux';
+import { useNavigate } from 'react-router-dom';
+import CoinsData from '../components/CoinsData';
+import Header from '../components/Header';
+import '../styles/CoinsList.css';
+
```

Comment:
The styling of the list on your Homepage does not conform to the alternating dual-color pattern of the provided [design template](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as well as the column layout of cells. Kindly note we as much as possible want your design to match the provided [template](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)). Therefore kindly ensure that:
- [ ] Your design **fits** perfectly on mobile screens
- [ ] The cards/ cells on the _Home page_ are **two columns** on mobile.
- [ ] The cells/ card of your list follow the **alternating dual-color pattern**. Each cell of the list item should have surrounding cells having **different (alternating) background color(s)**.

**Your design:**        |  **The original design:**
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/12745474/204087839-94b66c87-5057-42c3-8b74-f904adbb2e55.png) |  ![image](https://user-images.githubusercontent.com/12745474/142270463-4d16b9d3-7557-430c-80eb-a7c40589a4a6.png)